students/toddschultz/session03/slicing_lab_exercise.py

```diff
@@ -0,0 +1,48 @@
+#with the first and last items exchanged.
+def exchanged_string(seq):
+    return seq[-1:] + seq[1:-1] + seq[:1]
+exchanged_string("abcdefghijklmnopqrstuvwxyz")
+
+def exchanged_tuple(seq):
+    return seq[-1:] + seq[1:-1] + seq[:1]
+exchanged_tuple(("airplane", "boat", "car", "donkey", "elevator", "frigate", "glider", "hovercraft", "inflatable"))
+
+#with every other item removed.
+def every_other_tuple(seq):
+    return seq[::2] 
+every_other_tuple(("apple", "banana", "Corn", "Date", "Elderberry", "Fig", "Grape", "Honeydew", "Imbe"))
+
+def every_other_string(seq):
+    return seq[::2] 
+every_other_string("123456789")
```

Comment:
for each task there should be a single function that handles any iterable (string, list, tuple); especially given that your code is identical, there is no need to duplicate the code with a different name.